packages/chips/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@material/react-chips",
+  "version": "0.0.0",
+  "description": "Material Components React Chips",
+  "license": "Apache-2.0",
+  "keywords": [
+    "mdc web react",
+    "material components react",
+    "material design",
+    "material chip",
+    "materialchip",
+    "material chips",
+    "materialchips"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web-react.git"
+  },
+  "dependencies": {
+    "@material/chips": "^0.36.0",
+    "@material/react-ripple": "^0.2.0",
+    "classnames": "^2.2.5",
```

Comment:
Do we have a way of making sure that minor version changes to these will not cause problems for the components?